src/integrationTest/resources/db/changelog/db.changelog-master.xml

```diff
@@ -0,0 +1,7 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <include file="db/changelog/db.changelog-main.xml" />
+  <include file="db/changelog/db.changelog-AM-823.xml" />
```

Comment:
AM-823 JIRA was used in role assignment service, We can update it with batch JIRA id.

Reply:
Hi Nitin, We should be using the same Liquibase scripts as the RAS, so that there is no discrepancy. 
The liquibase scripts are being used for Embedded postgres only.